packages/tables/src/Table/Concerns/CanSortRecords.php

```diff
@@ -55,9 +49,15 @@ public function getSortableVisibleColumn(string $name): ?Column
         return $column;
     }
 
-    public function getDefaultSortColumn(): ?string
```

Comment:
For BC, we can keep this method and inside `return is_string($this->defaultSort) ? $this->defaultSort : null`

Reply:
Please mark it as `@deprecated` with a replacement like the other places we do that